docs/install/core-install.rst

```diff
@@ -0,0 +1,201 @@
+
+      .. code-block:: shell
+
+         tar xfz omniperf-v{{ config.version }}.tar.gz
+         cd omniperf-v{{ config.version }}
+
+#. Next, install Python dependencies and complete the Omniperf configuration and
+   install process.
+
+   .. datatemplate:nodata::
+
+      .. code-block:: shell
+
+         # define top-level install path
+         export INSTALL_DIR=<your-top-level-desired-install-path>
+
+         # install python deps
+         python3 -m pip install -t ${INSTALL_DIR}/python-libs -r requirements.txt
+
+         # configure Omniperf for shared install
+         mkdir build
+         cd build
+         cmake -DCMAKE_INSTALL_PREFIX=${INSTALL_DIR}/{{ config.version }} \
+                 -DPYTHON_DEPS=${INSTALL_DIR}/python-libs \
+                 -DMOD_INSTALL_PATH=${INSTALL_DIR}/modulefiles ..
+
+         # install
+         make install
+
+   .. tip::
+
+      You might need to ``sudo`` the final installation step if you don't have
+      write access for the chosen installation path.
+
+#. Upon successful installation, your top-level installation directory should
+   look like this.
+
+   .. datatemplate:nodata::
+
+      .. code-block:: shell
+
+         $ ls $INSTALL_DIR
+         modulefiles  {{ config.version }}  python-libs
+
+.. _core-install-modulefiles:
+
+Execution using modulefiles
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The installation process includes the creation of an environment modulefile for
+use with `Lmod <https://lmod.readthedocs.io>`_. On systems that support Lmod,
+you can register the Omniperf modulefile directory and setup your environment
+for execution of Omniperf as follows.
+
+.. datatemplate:nodata::
+
+   .. code-block:: shell
+
+      $ module use $INSTALL_DIR/modulefiles
+      $ module load omniperf
+      $ which omniperf
+      /opt/apps/omniperf/{{ config.version }}/bin/omniperf
+
+      $ omniperf --version
+      ROC Profiler:   /opt/rocm-5.1.0/bin/rocprof
+
+      omniperf (v{{ config.version }})
+
+.. tip::
+
+   If you're relying on an Lmod Python module locally, you may wish to customize
+   the resulting Omniperf modulefile post-installation to include extra
+   module dependencies.
+
+Execution without modulefiles
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To use Omniperf without the companion modulefile, update your ``PATH``
+settings to enable access to the command line binary. If you installed Python
+dependencies in a shared location, also update your ``PYTHONPATH``
+configuration.
+
+.. datatemplate:nodata::
+
+   .. code-block:: shell
+
+      export PATH=$INSTALL_DIR/{{ config.version }}/bin:$PATH
+      export PYTHONPATH=$INSTALL_DIR/python-libs
+
+.. _core-install-rocprof-var:
+
+ROCProfiler
+-----------
+
+Omniperf relies on :doc:`ROCProfiler <rocprofiler:index>`'s ``rocprof`` binary
+during the profiling process. Normally, the path to this binary is detected
+automatically, but you can override the path by the setting the optional
+``ROCPROF`` environment variable.
```

Comment:
I think it would be important to note that setting ROCPROF may be essential to running omniperf successfully. 
For example, MI300 will require `export ROCPROC=rocprofv2` otherwise you'll see the following error:   `ERROR gfx942 is not enabled in rocprofv1. Available profilers include: ['rocprofv2']`

Reply:
We should probably just relax this requirement now.  My understanding from the profiler team is gfx942 is fully supported in v1 at this point.

Reply:
Agreed. I've filed a ticket to track progress on getting this change made:
https://github.com/ROCm/omniperf/issues/388

@nartmada could you assign this to a developer as you see fit? Thank you.